submitit/snapshot.py

```diff
@@ -0,0 +1,38 @@
+import fnmatch
+import os
+import shutil
+from subprocess import check_output
+
+
+def run_cmd(str_args, **kwargs):
+    return check_output(str_args, shell=True, **kwargs).decode("utf-8").strip()
```

Comment:
`subprocess.check_output(str_args, encoding="utf-8", **kwargs).strip()` 

I'd rather not make `shell=True` implicit. We only want it when we have a pipe.